test/performance/cpu.gnuplot

```diff
@@ -0,0 +1,24 @@
+# gnuplot -e "titel='CPU during GET'; filename='merge/get-cpu.csv'; outfile='get-cpu.png'" test/performance/cpu.gnuplot
+# gnuplot -e "titel='CPU during POST'; filename='merge/create-cpu.csv'; outfile='create-cpu.png'" test/performance/cpu.gnuplot
+set datafile separator ','
+
+set title titel
```

Comment:
again, is `titel` intentional?

```suggestion
set title title
```

Reply:
Not sure if this will work, did you try it?

Reply:
nope, I just assumed this was a typo

Reply:
Changed it to `name` so its cleared and do not resemble a typo.